readme.md

```diff
@@ -292,6 +292,16 @@ $faker->optional($weight = 0.5, $default = false)->randomDigit; // 50% chance of
 $faker->optional($weight = 0.9, $default = 'abc')->word; // 10% chance of 'abc'
 ```
 
+Since v1.5 this method will accept a percentage like the `boolean()` method.
```

Comment:
I think you should just replace the old doc with the new preferred way, which is the integer way. As for the BC break, it's very minimal. I'll just mention it in the release post.